packages/cashier/src/components/cashier-container/virtual/virtual.tsx

```diff
@@ -46,12 +45,7 @@ const Virtual = observer(() => {
                                 key={1}
                                 className='virtual__account-switch-text'
                                 onClick={() => {
-                                    if (is_pre_appstore) {
-                                        history.push(routes.trade);
```

Comment:
why removing this `history.push(routes.trade)`?

Reply:
Because initially they were going to add the account switcher to cashier, so we dont need to redirect them to dtrader.